Makefile

```diff
@@ -34,6 +34,7 @@ codecov: .coverage
 lint:
 	@pip install -q tox
 	tox -e lint
+	$(GULP) lint
```

Comment:
I followed the pattern of the existing `test` rule in running JS + Python linting in the same rule instead of separating them.